### PR TITLE
Trying to have imports as a python dict

### PR DESCRIPTION
**Pull request by syrusakbary**
Trying to have imports as a python dict

@Amanieu

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -8,4 +8,13 @@ members = [
...
+]
+
+# [patch.crates-io]
+# wasmer = { path = "../wasmer/lib/api" }
+# wasmer-wasi = { path = "../wasmer/lib/wasi" }
+# wasmer-types = { path = "../wasmer/lib/types" }
+# wasmer-compiler-cranelift = { path = "../wasmer/lib/compiler-cranelift" }
+# wasmer-compiler-llvm = { path = "../wasmer/lib/compiler-llvm" }
+# wasmer-compiler-singlepass = { path = "../wasmer/lib/compiler-singlepass" }
+# wasmer-compiler = { path = "../wasmer/lib/compiler" }
```
Comment (Amanieu) [inline Cargo.toml:20]: Don't commit this. Also undo the changes to Cargo.lock.

Comment (syrusakbary) [reply Cargo.toml]: The `Cargo.lock` changes I think I forgot to commit from `master`, so they were dangling (not part of this PR)

**Review by Amanieu — APPROVED** [review]
